composer.json

```diff
@@ -55,6 +66,8 @@
     "autoload-dev": {
         "psr-4": {
             "SprykerEcoTest\\": "tests/SprykerEcoTest/",
+            "Orm\\": "tests/app/src/Orm/",
+            "Generated\\Shared\\": "tests/app/src/Generated/Shared",
```

Comment:
This  should not be here afaik, but instead autoloader in bootstrap should be used as documented.

Reply:
Sure.